src/operator-manual/metrics/01-quickstart.md

```diff
@@ -0,0 +1,235 @@
+rules.
+
+Let's create a `kube-prometheus-stack-values.yaml` file to configure the `kube-prometheus-stack`
+Helm chart values with the following contents:
+
+```yaml
+prometheus:
+  additionalServiceMonitors:
+    - name: kubewarden
+      selector:
+        matchLabels:
+          app: kubewarden-policy-server-default
+      namespaceSelector:
+        matchNames:
+          - kubewarden
+      endpoints:
+        - port: metrics
+          interval: 10s
+```
+
+The `prometheus-operator` deployed as part of this Helm chart defines the concept of [Service
+Monitors](https://github.com/prometheus-operator/prometheus-operator/blob/master/Documentation/design.md#servicemonitor),
+used to declaratively define what services should be monitored by Prometheus.
+
+In our case, we are adding a Service monitor targeting the `kubewarden` namespace, for services that
+match labels `app=kubewarden-policy-server-ha-policy-server`. This way, the Prometheus Operator is
+able to inspect what `endpoints` are tied to services matching this conditions. The operator will
+then take care of generating a valid configuration file for Prometheus, and reload Prometheus
+automatically after updating its configuration file.
+
+```console
+helm install --wait --create-namespace --namespace prometheus --values kube-prometheus-stack-values.yaml prometheus prometheus-community/kube-prometheus-stack
+```
+
+## Install Kubewarden
+
+We can now install Kubewarden in the recommended way with the Helm chart.
+
+> **Note well:** cert-manager and OpenTelemetry are a requirement of Kubewarden, but we've already
+> installed them in a previous section of this book.
+
+As a first step, we have to add the Helm repository that contains Kubewarden:
+
+```console
+helm repo add kubewarden https://charts.kubewarden.io
+```
+
+Then we have to install the Custom Resource Definitions (CRDs) defined by
+Kubewarden:
+
+```console
+helm install kubewarden-crds kubewarden/kubewarden-crds
+```
+
+Now we can deploy the rest of the Kubewarden stack. The official helm chart will create a
+PolicyServer named `default`. The default Helm chart values are already set up in a way that metrics
+are exposed for this default Policy Server. The relevant Helm Chart values are:
+
+```yaml
+policyServer
+  telemetry:
+    metrics:
+      port: 8080
+```
+
+Note that the default `values.yaml` for this Helm Chart are already set to this values. There is no
+need set this values if you don't want to change anything else.
+
+We can install the helm chart:
+
+```console
+helm install --wait --create-namespace --namespace kubewarden kubewarden-controller kubewarden/kubewarden-controller
+```
+
+This leads to the creation of the `default` instance of `PolicyServer`:
+
+```console
+kubectl get policyservers.policies.kubewarden.io
+NAME      AGE
+default   3m7s
+```
+
+By default, this policy server will have metrics enabled.
+
+## Enforcing a policy
```

Comment:
I have some doubts about this section. I would skip it tbh

Reply:
I agree.

Reply:
Should we skip it on tracing too? I wanted to show a way to make the counters go up and be able to inspect how they grow.

Reply:
Would it be enough just explaining in a paragraph what is expected to happen?
To be fair, I have no problem with it as an example after the whole set up, but the maintenance of the examples without the single source of truth may get annoying.

Reply:
I agree :100:% with what @viccuad said.